ebill_paynet/models/paynet_service.py

```diff
@@ -0,0 +1,223 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+import os
+
+from lxml import etree
+from odoo import api, fields, models
+from ..components.api import PayNetDWS
+from odoo.exceptions import UserError
+from zeep.exceptions import Fault
+
+SYSTEM_PROD_URL = 'https://dws.paynet.ch/DWS/DWS'
+SYSTEM_TEST_URL = 'https://dws-test.paynet.ch/DWS/DWS'
```

Comment:
I suggest to put that in `ir.config_parameter`?

Reply:
I would say not worth it because this standard paynet urls they change them, it changes for all their customer I guess